docs/index.md

```diff
@@ -238,15 +242,8 @@ If the bucket is in another AWS account, the bucket policy will need to grant ac
       "Principal": {
         "AWS": "arn:aws:iam::123456789012:user/YOUR_USER"
       },
-      "Action": [
-        "s3:ListBucket",
-        "s3:GetObject",
-        "s3:GetObjectVersion"
-      ],
-      "Resource": [
-        "arn:aws:s3:::test-bucket1",
-        "arn:aws:s3:::test-bucket1/*"
-      ]
+      "Action": ["s3:ListBucket", "s3:GetObject", "s3:GetObjectVersion"],
```

Comment:
Why was this changed? I'd prefer to leave it the same (especially if other index docs use the current format)

Reply:
Due to my VS code setup, it got updated automatically after I saved the file. Reverted back.